assembl/tests/views_tests/test_graphql_vote_session.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+
+from assembl.graphql.schema import Schema as schema
+
+
+def snake_to_camel(string):
+    # https://stackoverflow.com/questions/19053707/converting-snake-case-to-lower-camel-case-lowercamelcase
+    components = string.split('_')
+    # We capitalize the first letter of each component except the first one
+    # with the 'title' method and join them together.
+    return components[0] + "".join(x.title() for x in components[1:])
+
+
+def assert_langstring_is_equal(langstring_name, fetched_model, source_model):
+    source_value = getattr(source_model, langstring_name).entries[0].value
+    fetched_value = fetched_model[snake_to_camel(langstring_name) + 'Entries'][0]['value']
+    assert source_value == fetched_value
+
+
+def assert_langstrings_are_equal(langstrings_names, fetched_model, source_model):
```

Comment:
maybe you could move these 3 helpers to `tests/utils` ?

Reply:
yes but I would need to modify them to be more generic instead of assuming there is only one entry

Reply:
ok, so they can stay here in the meanwhile and if someone needs something similar he will refactor/move the helpers

Reply:
The `snake_to_camel` is generic enough to be moved, tbf. But, keep these here for now. We'll make this more generic as more tests are being written.